source/agora/test/Base.d

```diff
@@ -954,6 +955,12 @@ public class TestNetworkManager : NetworkManager
                "' without first creating it");
     }
 
+    ///
+    public override void startStatsServer()
+    {
+        stats_server = new StatsServer(0);
```

Comment:
In Linux, port `0` means "pick whatever". I'm wondering, why does it need to be a different server / port though ? Why not simply expose it in the API (or make an API just for it) ?

Reply:
would you prefer if the user specify -1 if he wants to disable startup up the metrics server?